src/index.ts

```diff
@@ -413,7 +348,10 @@ function noExternalInertiaHelpers(config: UserConfig): true|Array<string|RegExp>
     /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
     /* @ts-ignore */
     const userNoExternal = (config.ssr as SSROptions|undefined)?.noExternal
-    const pluginNoExternal = ['laravel-vite-plugin']
+    const pluginNoExternal = [
+        'laravel-vite-plugin/inertia-helpers',
+        '@inertiajs/server'
```

Comment:
This is likely something we need to push upstream and fix in inertia, but putting it here so I can move on.